src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const empoyeeslist = document.querySelector('ul');
+
+function sortList(list) {
+  function moneyToNumber(string) {
+    const filtredString = string.replace(/\$/g, '').replace(/,/g, '');
+
+    return filtredString;
```

Comment:
The `moneyToNumber` function should convert the filtered string to a number. Consider using `parseFloat(filtredString)` to ensure it returns a numeric value.